tests/baselines/reference/docker/vue-next.log

```diff
@@ -90,49 +75,27 @@ created packages/reactivity/dist/reactivity.global.prod.js in ?s
 /vue-next/packages/reactivity/src/index.ts → packages/reactivity/dist/reactivity.esm.prod.js...
 created packages/reactivity/dist/reactivity.esm.prod.js in ?s
 /vue-next/packages/runtime-core/src/index.ts → packages/runtime-core/dist/runtime-core.esm-bundler.js...
-created packages/runtime-core/dist/runtime-core.esm-bundler.js in ?s
-/vue-next/packages/runtime-core/src/index.ts → packages/runtime-core/dist/runtime-core.cjs.js...
-created packages/runtime-core/dist/runtime-core.cjs.js in ?s
-/vue-next/packages/runtime-core/src/index.ts → packages/runtime-core/dist/runtime-core.cjs.prod.js...
-created packages/runtime-core/dist/runtime-core.cjs.prod.js in ?s
-/vue-next/packages/runtime-dom/src/index.ts → packages/runtime-dom/dist/runtime-dom.esm-bundler.js...
-created packages/runtime-dom/dist/runtime-dom.esm-bundler.js in ?s
-/vue-next/packages/runtime-dom/src/index.ts → packages/runtime-dom/dist/runtime-dom.cjs.js...
-created packages/runtime-dom/dist/runtime-dom.cjs.js in ?s
-/vue-next/packages/runtime-dom/src/index.ts → packages/runtime-dom/dist/runtime-dom.global.js...
-created packages/runtime-dom/dist/runtime-dom.global.js in ?s
-/vue-next/packages/runtime-dom/src/index.ts → packages/runtime-dom/dist/runtime-dom.esm.js...
-created packages/runtime-dom/dist/runtime-dom.esm.js in ?s
-/vue-next/packages/runtime-dom/src/index.ts → packages/runtime-dom/dist/runtime-dom.cjs.prod.js...
-created packages/runtime-dom/dist/runtime-dom.cjs.prod.js in ?s
-/vue-next/packages/runtime-dom/src/index.ts → packages/runtime-dom/dist/runtime-dom.global.prod.js...
-created packages/runtime-dom/dist/runtime-dom.global.prod.js in ?s
-/vue-next/packages/runtime-dom/src/index.ts → packages/runtime-dom/dist/runtime-dom.esm.prod.js...
-created packages/runtime-dom/dist/runtime-dom.esm.prod.js in ?s
-/vue-next/packages/runtime-test/src/index.ts → packages/runtime-test/dist/runtime-test.global.js...
-created packages/runtime-test/dist/runtime-test.global.js in ?s
-/vue-next/packages/runtime-test/src/index.ts → packages/runtime-test/dist/runtime-test.global.prod.js...
-created packages/runtime-test/dist/runtime-test.global.prod.js in ?s
-/vue-next/packages/server-renderer/src/index.ts → packages/server-renderer/dist/server-renderer.cjs.js...
-[!] (plugin rpt2) Error: /vue-next/packages/vue/src/devCheck.ts(2,11): semantic error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
-packages/vue/src/devCheck.ts
-Error: /vue-next/packages/vue/src/devCheck.ts(2,11): semantic error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
+[!] (plugin rpt2) Error: /vue-next/packages/runtime-core/src/apiOptions.ts(137,18): semantic error TS2344: Type 'Function & T[key]["get"]' does not satisfy the constraint '(...args: any) => any'.
```

Comment:
Seems correct, `ReturnType<Function>` isn't allowed because Function is callable but doesn't have any signatures. I'm not sure why we didn't complain about this before.

It's pretty easy to fix: change

```ts
export type ExtractComputedReturns<T extends any> = {
  [key in keyof T]: T[key] extends { get: Function }
    ? ReturnType<T[key]['get']>
    : ReturnType<T[key]>
}
```

to

```ts
export type ExtractComputedReturns<T extends Record<string, (...args:any[]) => any>> = {
    [key in keyof T]: T[key] extends { get: (...args:any[]) => any }
    ? ReturnType<T[key]['get']>
    : ReturnType<T[key]>
}
```

I'll either contact the vue team or send a PR.